Small change ingestion mechanisms Documentation (#13630)
DataDog/documentation: content/en/tracing/trace_ingestion/mechanisms.md

To each span ingested, there is attached a unique **ingestion reason** referring to one of the mechanisms described in this page. Usage metrics `datadog.estimated_usage.apm.ingested_bytes` and `datadog.estimated_usage.apm.ingested_spans` are tagged by `ingestion_reason`.

## Head-based sampling

The default sampling mechanism is called _head-based sampling_. The decision of whether to keep or drop a trace is made at the very beginning of the trace, at the start of the [root span][2]. This decision is then propagated to other services as part of their request context, for example as an HTTP request header.

Because the decision is made at the beginning of the trace and then conveyed to all parts of the trace, the trace is guaranteed to be kept or dropped as a whole.

> Sampling rates for head-based sampling can be set up in two places:

Reviewer: ```suggestion
You can set sampling rates for head-based sampling in two places:
```

## Error and rare traces

For traces not caught by the head-based sampling, **Agent** mechanisms make sure that critical and diverse traces are kept and ingested. These two samplers keep a diverse set of traces by catching all combinations of a predetermined set of tags:

- **Error traces**: Sampling errors is important for providing visibility on potential system failures.
- **Rare traces**: Sampling rare traces allows you to keep visibility on your system as a whole, by making sure that low-traffic services and resources are still monitored.

> **Note**: Error and rare samplers will be ignored for services for which you set [library sampling rules](#in-tracing-libraries-user-defined-rules)

Reviewer: ```suggestion
**Note**: Datadog ignores error and rare samplers for services for which you set [library sampling rules](#in-tracing-libraries-user-defined-rules).
```

> **Note**: Datadog ignores error and rare samplers for services for which you set [library sampling rules](#in-tracing-libraries-user-defined-rules).

Author: ```suggestion
**Note**: Error and rare samplers are ignored for services for which you set [library sampling rules](#in-tracing-libraries-user-defined-rules).
```
